spec/requests/issues_spec.rb

```diff
@@ -2,9 +2,48 @@
 
 RSpec.describe "Issues", type: :request do
   describe "GET /issues" do
-    it "works! (now write some real specs)" do
+    let(:user) {create(:user)}
+    let!(:issue) {create(:issue, user_id: user.id, principal: "Starbucks")}
+    let!(:issue2) {create(:issue, user_id: user.id)}
+    let!(:issue3) {create(:issue, user_id: user.id)}
+    it "returns all of the issues" do
```

Comment:
Maybe this should be a context?
`context "returns all of the issues"`

Reply:
i think you only need a context if you are grouping specs based on different situations, there is only one spec right now so a context isnt needed